app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -92,6 +92,11 @@ export const FixedSessionsMapCtrl = (
         $scope.sessions.fetch();
       });
 
+      elmApp.ports.toggleStreaming.subscribe(isStreaming => {
+        params.update({ data: { isStreaming: isStreaming }});
```

Comment:
`isStreaming: isStreaming` == `isStreaming` if you prefer

Reply:
is this better?

Reply:
I like it

Reply:
why do you like it better? what's the +?

Reply:
I guess less code? I don't have a strong opinion

Reply:
hmmm then I guess I didn't get the change, I'll ask offline